SPEC.md

```diff
@@ -700,8 +702,8 @@ JSON Schema for the Image Manifest (app image manifest, ACI manifest), conformin
     },
     "dependencies": [
         {
-            "app": "example.com/reduce-worker-base",
-            "imageID": "sha512-...",
+            "name": "example.com/reduce-worker-base",
+            "id": "sha512-...",
```

Comment:
sorry, I already started down this path but forgot to put up a PR - https://github.com/jonboulle/spec/commit/e343fa9f41ab0822001fe02bfde469872e8de6de